From bf116697c55c7c6a199782252f179f90b34a7072 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Tue, 15 May 2018 18:15:54 +0000
Subject: [PATCH] OS-6958 bad comparison introduced with OS-6630 Reviewed by:
 John Levon <john.levon@joyent.com> Reviewed by: Josh Wilsdon
 <jwilsdon@joyent.com> Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 src/vm/sbin/vmadmd.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/vm/sbin/vmadmd.js b/src/vm/sbin/vmadmd.js
index 4f1974e2..4c1f7338 100755
--- a/src/vm/sbin/vmadmd.js
+++ b/src/vm/sbin/vmadmd.js
@@ -1126,7 +1126,7 @@ function updateZoneStatus(ev)
         }
 
         // don't handle transitions other than provisioning for non-kvm/bhyve
-        if (['bhyve', 'kvm'].indexOf(vmobj.brand) !== -1) {
+        if (['bhyve', 'kvm'].indexOf(vmobj.brand) === -1) {
             log.trace('doing nothing for ' + ev.zonename + ' transition '
                 + 'because brand "' + vmobj.brand
                 + '" is not "kvm" or "bhyve"');
-- 
2.21.0

